test/blocks/feed/feed.decorated.html

```diff
@@ -1 +1 @@
\ No newline at end of file
+<div class="cards block" data-block-name="cards" data-block-status="loaded"><ul><li><a href="undefined"><div class="cards-card-image"><picture><source type="image/webp" srcset="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=webply&amp;optimize=medium"><img loading="lazy" alt="" src="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=jpeg&amp;optimize=medium"></picture></div><div class="cards-card-body"><div><div></div><div class="title">4 - Sunstar Engineering Americas wins the Mazda Excellence...</div></div></div></a></li><li><a href="undefined"><div class="cards-card-image"><picture><source type="image/webp" srcset="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=webply&amp;optimize=medium"><img loading="lazy" alt="" src="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=jpeg&amp;optimize=medium"></picture></div><div class="cards-card-body"><div><div></div><div class="title">3 - Sunstar Engineering Americas wins the Mazda Excellence...</div></div></div></a></li><li><a href="undefined"><div class="cards-card-image"><picture><source type="image/webp" srcset="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=webply&amp;optimize=medium"><img loading="lazy" alt="" src="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=jpeg&amp;optimize=medium"></picture></div><div class="cards-card-body"><div><div></div><div class="title">2 - Sunstar Engineering Americas wins the Mazda Excellence...</div></div></div></a></li><li><a href="undefined"><div class="cards-card-image"><picture><source type="image/webp" srcset="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=webply&amp;optimize=medium"><img loading="lazy" alt="" src="/media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&amp;format=jpeg&amp;optimize=medium"></picture></div><div class="cards-card-body"><div><div></div><div class="title">Type is Newsroom. Category is Corporate.</div></div></div></a></li></ul></div>
```

Comment:
Can you please remove these two
 🚧 404 network requests:
    - media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=2000&format=webply&optimize=medium
    - media_102f1ae6e980ab88e16787e84357894cec4e07a16.jpeg?width=750&format=webply&optimize=medium